spdx/writers/rdf.py

```diff
@@ -208,7 +209,7 @@ def create_file_node(self, doc_file):
         """
         Create a node for spdx.file.
         """
-        file_node = BNode()
+        file_node = URIRef('http://www.spdx.org/files#' + str(doc_file.spdx_id))
```

Comment:
You could prefer `.format` when possible rather than plain string concatenation.... e.g. something like:
`'http://www.spdx.org/files#{id}'.format(id=str(doc_file.spdx_id))`

Reply:
Roger that.